e2e/do_test/do_case_group/test_params/test.yml

```diff
@@ -0,0 +1,10 @@
+query: >
+    CALL do.case([TRUE, "MATCH (n {id: $id}) RETURN n;"],
```

Comment:
Is it possible to have multiple CALLs within one query? Like you do CALL do.case bla bla you get some output values and can those values be send further into other query modules? Not sure, just a question.

Reply:
That’s possible in general :slightly_smiling_face: